.github/workflows/cmake.yml

```diff
@@ -129,7 +132,7 @@ jobs:
       - name: upload bundle artifacts
         uses: actions/upload-artifact@v3
         with:
-          name: ${{ runner.os }}-${{ matrix.arch }}
+          name: ${{ matrix.target }}
           path: |
             ./build/bundle/ziti-edge-tunnel-*.zip
```

Comment:
We can exclude the test binaries with a negation:

https://github.com/openziti/ziti-tunnel-sdk-c/pull/623/files#diff-cdd48abbd3eb8d1c54077449fc74a8de1f29805d2be5d8e5232b7aab76ea7a6fR148

Reply:
better yet, do not bundle ziti-edge-tunnel-test at all